### DISCUSSION TITLE
Apple Books提供AI語音功能，可將書籍轉成有聲讀物

### DISCUSSION THREAD

**Discussion by Xuewen628** [1101-GS3073-C/Python-Community-GS3073-C#633] [General]
現在的科技已經進步得能帶給人類更多的幫助了，這種科技的發展能讓盲人有更多吸收知識的機會
希望以後能有更多類似的發明
https://www.ithome.com.tw/news/155045

**Comment by awehao** [comment]
這好像就是有聲書诶，還以為有什麼重大的突破，還是其實我誤解了，我覺得以現在的處理速度，還不足以站穩主要的讀書方式，在獲取之是這一塊，我覺得還是腳踏實地會更好
